src/coreComponents/constitutive/permeability/DisplacementDependentPermeability.hpp

```diff
@@ -0,0 +1,173 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file DisplacementDependentPermeability.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_PERMEABILITY_DISPLACEMENTDEPENDENTPERMEABILITY_HPP_
+#define GEOSX_CONSTITUTIVE_PERMEABILITY_DISPLACEMENTDEPENDENTPERMEABILITY_HPP_
+
+#include "constitutive/permeability/PermeabilityBase.hpp"
+
+
+namespace geosx
+{
+namespace constitutive
+{
+
+class DisplacementDependentPermeabilityUpdate : public PermeabilityBaseUpdate
+{
+public:
+
+  DisplacementDependentPermeabilityUpdate( arrayView3d< real64 > const & permeability,
+                                           arrayView3d< real64 > const & dPerm_dPressure,
+                                           arrayView3d< real64 > const & dPerm_dDisplacement,
+                                           real64 const shearDispThreshold,
+                                           real64 const maxPermMultiplier,
+                                           arrayView3d< real64 > const & iniPermeability )
+    : PermeabilityBaseUpdate( permeability, dPerm_dPressure ),
+    m_dPerm_dStrain( dPerm_dDisplacement ),
```

Comment:
this is confusing ... the primary unknown here is the displacement *jump* across the fault/fracture (not the displacement or the strain).

Reply:
Right, just change it to the dispJump

Reply:
As only shear displacement jump is considered for the permeability, I propose to specify it with a "shear" or "slip" term.